docs/contributing/contributing-code/contributing-code-tests/tests-images-pushtoghcr.md

```diff
@@ -0,0 +1,64 @@
+
+## Using Standard Images in Tests
+
+When writing tests for our applications, it's common to use standard images for resources like Redis, Mongo, and others. By default, specifying an image with `redis:latest` or similar tags will pull the image from Docker Hub. This behavior is due to Docker's default configuration, which automatically searches the Docker Hub registry for images when no other registry is specified. 
+
+ We are transitioning to using images hosted in our public GitHub Container Registry (GHCR) repository. This change ensures more reliable access to the images needed for our tests and will help reduce potential issues caused by external dependencies.
+
+### New Image Pull Guideline
+
+Moving forward, we will pull standard test images from our public GHCR repository instead of Docker Hub. 
+
+For example, instead of using:
+```yaml
+image: 'redis:latest'
+```
+We should now use:
+```yaml
+image: ghcr.io/radius-project/mirror/redis:latest
```

Comment:
What is the 'mirror' desciptor meant to be?

Reply:
this was suggested by @youngbupark [here](https://github.com/radius-project/radius/pull/7530#discussion_r1577312961)

Reply:
Just checked one more time and we don't have Redis latest tag in our packages: https://github.com/orgs/radius-project/packages/container/package/mirror%2Fredis.

Reply:
All other tests updated for this refer to redis:6.2. This test works as expected with the label. Yes, we do not have the image with latest tag. if it is really needed for this test, I'll need to request rights from maintainers to add it to the image repo.